IdentityCore/src/oauth2/aad_base/MSIDAADWebviewFactory.m

```diff
@@ -134,24 +141,38 @@ - (MSIDWebviewResponse *)oAuthResponseWithURL:(NSURL *)url
             return nil;
         }
     }
+    
+    // Try to create JIT troubleshooting response
+    MSIDJITTroubleshootingResponse *jitResponse = [[MSIDJITTroubleshootingResponse alloc] initWithURL:url context:context error:nil];
+    if (jitResponse && error)
```

Comment:
Why are you ignoring jit response if there's no error?

Reply:
I put the check for `error` in the incorrect place. Updated, thanks